store/src/commonMain/kotlin/org/mobilenativefoundation/store/store5/impl/FetcherController.kt

```diff
@@ -87,18 +92,22 @@ internal class FetcherController<Key : Any, Input : Any, Output : Any>(
                  */
                 piggybackingDownstream = true,
                 onEach = { response ->
-                    response.dataOrNull()?.let { input ->
-                        sourceOfTruth?.write(key, input)
+                    response.dataOrNull()?.let { networkRepresentation ->
+                        val input =
+                            networkRepresentation as? CommonRepresentation ?: converter?.fromNetworkRepresentationToCommonRepresentation(networkRepresentation)
```

Comment:
maybe move the noop logic to the converter

Reply:
Will address separately tomo #498